modules/planning/tasks/deciders/open_space_fallback_decider.cc

```diff
@@ -30,6 +30,27 @@ using apollo::common::math::Vec2d;
 OpenSpaceFallbackDecider::OpenSpaceFallbackDecider(const TaskConfig& config)
     : Decider(config) {}
 
+bool QuardraticFormulaLowerSolution(
+  const double& a, const double& b, const double& c, double* sol) {
```

Comment:
https://github.com/ApolloAuto/apollo/blob/master/docs/technical_tutorial/best_coding_practice.md
#5.2: For input scalas, just pass by value, which gives better locality and thus performance.